warn-stranded.lua

```diff
@@ -0,0 +1,204 @@
+-- Detects and alerts when a citizen is stranded
+-- by Azrazalea
+-- Logic heavily based off of warn-starving
+-- GUI heavily based off of autobutcher
+-- Thanks myk002 for telling me about pathability groups!
+--@ module = true
+
+local gui = require 'gui'
+local utils = require 'utils'
+local widgets = require 'gui.widgets'
+
+local function clear()
+   dfhack.persistent.delete('warnStrandedIgnore')
+end
+
+local args = utils.invert({...})
+if args.clear then
+   clear()
+end
```

Comment:
should move this below the `isMapLoaded()` check so you don't try to clear persistent data in a world that doesn't exist yet